internal/service/proto/proto.proto

```diff
@@ -1,23 +1,23 @@
 syntax = "proto3";
-package reclaimspace;
+package proto;
 
-option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/service/reclaimspace";
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/service/proto";
```

Comment:
the filename should stay `reclaimspace.proto`. The package can just be `proto`. When fencing is added, the `fence.proto` file (and generated `.go` files) should then be included in the same `proto` package.

I am not sure if the `service` part in the directory structure will be beneficial. Maybe it is cleaner to drop that?